### DISCUSSION TITLE
Dynamic AUC and time dependent concordance

### DISCUSSION THREAD

**Discussion by tommyvcc** [autonlab/auton-survival#101] [Q&A]
Thank you so much for the hard work
I have one question regarding the AUC metric.
Is it a "cumulative dynamic AUC " or " incident dynamic AUC" ?
For example, if the time intervals are 0 to 1 , 1 to 2 and 2 to 3.
While the AUC is 0.6, 0.7, 0.8
Does this mean that the AUC for the period 0 till 3 is 0.8?
Or the AUC for the period 2 to 3 is 0.8?

**Comment by Frederik-Hartmann** [comment]
I am not completely following your questions, but I hope this helps anyway:

In the current version auton-survival uses scikit-survivals cumulative_dynamic_auc
https://github.com/autonlab/auton-survival/blob/5dde465f7223601717abddc1d075e837707c403b/auton_survival/metrics.py#L31
https://github.com/autonlab/auton-survival/blob/5dde465f7223601717abddc1d075e837707c403b/auton_survival/metrics.py#L258-L265

In the docs of scikit-survival https://scikit-survival.readthedocs.io/en/stable/api/generated/sksurv.metrics.cumulative_dynamic_auc.html you can find more information.

Coming back to your question. If the time points are 0, 1 and 2. You will get the cumulative_dynamic_auc at these specific time points. e.g. the cumulative dynamic AUC at time point 1 would be 0.6 in your example.  If I am not mistaken the cumulative dynamic AUC is not calculated over a time range, but rather compares the test samples to your time point.
